src/variables.json

```diff
@@ -71,19 +71,9 @@
   "white": "#fff",
   "transparent": "transparent",
 
-  "disabled-interactive-color": "rgba(127, 127, 127, 0.1)",
-  "disabled-interactive-color-dark": "rgba(127, 127, 127, 0.25)",
-
-  "inactive-text-color": "rgba(64, 64, 64, 0.45)",
-
-  "text-color": "rgba(0, 0, 0, 0.75)",
-
-  "default-button-color": "#448ee4",
-  "default-button-color-dark": "#346db0",
-
-  "default-form-color-light": "#ccc",
-  "default-form-color": "#666",
-  "default-form-color-dark": "#2d2d2d",
+  "disabled10": "rgba(158, 163, 167, 0.1)",
+  "disabled25": "rgba(158, 163, 167, 0.25)",
+  "disabled75": "rgba(158, 163, 167, 0.75)",
```

Comment:
This color is exactly 50% between gray and gray light. It was picked because it's legible against both dark and light backgrounds unlike any other gray colors in the palette.